upgrading/upgrading_known_issues.adoc

```diff
@@ -48,181 +45,25 @@ fatal: [test.linux.lan]: FAILED! => {
-STDOUT:
-
-
-error: <1>
-oauthclientauthorizations/user1:system:serviceaccount:namespace1:jenkins : OAuthClientAuthorization "user1:system:serviceaccount:namespace1:jenkins" is invalid: clientName: Internal error: system:serviceaccount:namespace1:jenkins has no redirectURIs; set serviceaccounts.openshift.io/oauth-redirecturi.<some-value>=<redirect> or create a dynamic URI using serviceaccounts.openshift.io/oauth-redirectreference.<some-value>=<reference>
-
-error: <2>
-oauthclientauthorizations/user2:system:serviceaccount:namespace2:jenkins : OAuthClientAuthorization "user2:system:serviceaccount:namespace2:jenkins" is invalid: clientName: Internal error: serviceaccounts "jenkins" not found
-summary: total=1002 errors=2 ignored=0 unchanged=1000 migrated=0
-info: to rerun only failing resources, add --include=oauthclientauthorizations
-error: 2 resources failed to migrate
-----
-<1> This error occurs when the service account based OAuth client has no valid redirect URIs.
-<2> This error occurs when the OAuth client has been deleted.
-
-The first error can be solved in one of two ways:
-
-- Annotate the service account with a valid redirect URI; see xref:../architecture/additional_concepts/authentication.adoc#redirect-uris-for-service-accounts[Redirect URIs for Service Accounts as OAuth Clients].
-
-Or:
-
-- Delete the service account in the offending namespace if it is not being used
-anymore:
-+
-----
-$ oc delete serviceaccount <service_account> -n <namespace>
-----
-
-For the second error, a script is provided to automate the following:
-
-- Get all `oauthclientauthorization` objects.
-- Check if the associated OAuth client exists; it can be either an OAuth client or
-a service account.
-- Delete the `oauthclientauthorization` object if the associated OAuth client does
-not exist.
-
 To resolve this issue:
 
-. Save the following script to a file, for example
-*_oauthclientauthorization.sh_*:
-+
-[source,shell]
-----
-for oa in $(oc get -o name oauthclientauthorization); do
-	if [[ "$oa" != *":system:serviceaccount:"* ]]; then
-		client=$(echo "$oa" | cut -d : -f 2)
-		query="$(oc get oauthclient $client 2>&1)"
-		echo "$query" | grep NotFound
-		if [ "$?" == "0" ]; then
-			echo "Not found: $query"
-			oc delete $oa
-		fi
-	fi
-done
-
-for oa in $(oc get -o name oauthclientauthorization); do
-	if [[ "$oa" == *":system:serviceaccount:"* ]]; then
-		ns=$(echo "$oa" | cut -d : -f 4)
-		sa=$(echo "$oa" | cut -d : -f 5)
-		# echo "Found: $oa -> ns:$ns  sa: $sa"
-		query="$(oc get sa $sa -n $ns 2>&1)"
-		echo "$query" | grep "NotFound"
-		if [ "$?" == "0" ]; then
-			echo "Missing sa: $sa"
-			oc delete "$oa"
-			echo "    Delete operation: $?"
-		fi
-	fi
-done
-----
-
-. Run the script while logged in to the CLI as a cluster administrator:
-+
-----
-$ bash oauthclientauthorization.sh
-----
+. Log in to the CLI as a cluster administrator and inspect the `Service` objects
+listed in the error. Edit the `.spec.ports` of the objects removing duplicate
```

Comment:
I would make "Edit..."  its own numbered step, so:

. Log in to the CLI as a cluster administrator and inspect the `Service` objects listed in the error.

. Edit the `.spec.ports` of the objects removing duplicate . . .